File: 5-Matemcatica/1161-Soma_de_Fatoriais-5/1161.py
while True:    
    try:    
        x1, x2, = input().split()
        x1=int(x1)
        x2=int(x2)
        for i in range(x1-1, 0, -1):
            x1*=i
        for i in range(x2-1, 0, -1):
            x2*=i
        if x1 ==0:
            x1=1
        if x2 ==0:
            x2=1
        print(x1+x2)
    except:
        break

File: 5-Matemcatica/1161-Soma_de_Fatoriais-5/1161.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        while (true){
            try{
                string[] nuns = Console.ReadLine().Split(' ');
                int nu1=int.Parse(nuns[0]);
                long nun1=(long)nu1;
                for(long i=nun1-1; i>0; i-=1){
                    nun1*=i;
                }
                int nu2=int.Parse(nuns[1]);
                long nun2 =(long)nu2;
                for(long i=nun2-1; i>0; i-=1){
                    nun2*=i;
                }
                if (nun1 ==0){
                    nun1=1;
                }
                if (nun2 ==0){
                    nun2=1;
                }
                long result=nun1+nun2;
                Console.WriteLine(result);
            }
            catch(Exception ex){
                break;
            }
        }
    }
}